commit d477cba26ddfd6e4e34a71870f5bf671be6b4e0e
Author: Steve Muckle <smuckle@google.com>
Date:   Tue Nov 24 13:23:18 2020 -0800

    eas: fix overflow of in TS_TO_USEC
    
    Value is currently truncated.
    
    Signed-off-by: Steve Muckle <smuckle@google.com>
    Change-Id: I81a4be3f4fa649d565fa333bbfe4149feac06f0c

diff --git a/testcases/kernel/sched/eas/util.h b/testcases/kernel/sched/eas/util.h
index 2433a53d6..7108dcbe9 100644
--- a/testcases/kernel/sched/eas/util.h
+++ b/testcases/kernel/sched/eas/util.h
@@ -13,7 +13,7 @@
 
 #define USEC_PER_SEC 1000000
 
-#define TS_TO_USEC(x) (x.usec + x.sec * USEC_PER_SEC)
+#define TS_TO_USEC(x) (x.usec + (unsigned long long)x.sec * USEC_PER_SEC)
 
 #ifndef SCHED_DEADLINE
 #define SCHED_DEADLINE 6
